### DISCUSSION TITLE
Number of Total Shop Sales

### DISCUSSION THREAD

**Discussion by InaD89** [etsy/open-api#1010] [Q&A] [answered]
Hi everyone,

I'm looking for a way to request the total sales of a shop. I understand that Etsy is not providing sale details for listings, but is there a way to get the total sales (or any other metric that would help me estimate the average sales of a listing)?

The Etsy website shows the total sales for each shop, so I'm guessing there's a way to request it. Is there a possible way to do it, or maybe a workaround?

Thank you!

**Comment by Gareth064** [comment, marked as answer]
Hey @InaD89 

If you are talking about a shop that your app is authenticated with, then you can call the getShop endpoint and it will return a `transactions_sold_count` value for that shop.

If you are trying to look at other peoples shops which aren't authenticated with your app, then try using findShops endpoint. According to Etsy documentation, it will also return this field as well.

**Reply by InaD89** [reply]
Wow, I was not able to find this field at all. Thanks a lot!
